scripts/AbinsModules/FrequencyPowderGenerator.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import (absolute_import, division, print_function)
+
+import numpy as np
+
+import AbinsConstants
```

Comment:
Lines such as this will fail with `absolute_import`. They can be fixed by slightly rewriting the line to say:
```
from . import AbinsConstants
```

This will need to be done in the remaining places after the removal of the `sys.path` updates that @Anders-Markvardsen has already mentioned.